datadog_checks_base/datadog_checks/base/utils/http.py

```diff
@@ -827,6 +827,11 @@ def __init__(self, config):
             self._fetch_options['client_id'] = self._client_id
             self._fetch_options['client_secret'] = self._client_secret
 
+        self.auth_params = config.get('auth_params', {})
+        if isinstance(self.auth_params, dict):
+            for key, value in self.auth_params.items():
```

Comment:
```suggestion
        self._auth_params = config.get('auth_params', {})
        if isinstance(self.._auth_params, dict):
            for key, value in self.._auth_params.items():
```

For consistency with the other attributes we have in this class

Reply:
Thanks for your comment, I tried to add tests for it yesterday but failed, can you share with me how to run a single unit test locally? @FlorentClarret

Reply:
Ref https://github.com/requests/requests-oauthlib/blob/master/requests_oauthlib/oauth2_session.py#L253

Reply:
Got it, thanks